code/src/App.js

```diff
@@ -1,12 +1,54 @@
 import React from 'react'
 import data from './data.json'
+import playlistdata from './stretch-goal.json'
+
+
+import { Albums } from 'components/Albums'
+import { Singles } from 'components/Singles'
+import { Playlists } from 'components/Playlists'
+import { Header } from 'components/Header'
+import { Footer } from 'components/Footer'
 
-console.log(data)
 
 export const App = () => {
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    
+  <div>
+      <Header />
+
+      <h2>New Albums</h2>
+
+      <section>
+        {data.albums.items.map((singleAlbum) => {
+          if (singleAlbum.album_type === "album")
+          return <Albums key={singleAlbum.id} articleDetails={singleAlbum}/>
+          else return null
+        })}
+      </section>
+
+      <h2>New Singles</h2>
+
+      <section>
+        {data.albums.items.map((singleSingle) => {
+          if (singleSingle.album_type === "single")
```

Comment:
There is a different way or doing this: instead of using "else return null", try -catch method.